.github/workflows/pkg-installer.yml

```
@@ -11,6 +11,10 @@ on:
   release:
     types:
       - published
+permissions:
+  contents: read # for code access
+  attestations: write # for actions/attest-build-provenance
+  id-token: write # for actions/attest-build-provenance
```

```suggestion
permissions: {}
```
Can we these permissions to the `build` job instead of the top level?